iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceMethodCommon.java

```diff
@@ -59,7 +59,6 @@ public static Collection inputs() throws Exception
 
     protected static String iotHubConnectionString = "";
 
-    protected static DeviceMethod methodServiceClient;
```

Comment:
Having a static device method client is an issue since each client only allows one method invocation at a time ([the method invocation method is synchronized](https://github.com/Azure/azure-iot-sdk-java/blob/72b92781a529daa7bf75dd4dc87892e1c775f59b/service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceMethod.java#L64))

In theory, if one test's method invocation takes 10 seconds, that adds a 10 second delay to all concurrently running tests, too. 

I removed this static client in favor of one client per test instance so that no test can interfere with a different test through this.